### PR TITLE
docs: update readme file

### PR DESCRIPTION
**Pull request by Fasping**
With this PR, I am proposing an update to the README.md file to enhance the user experience and provide additional information. I believe these improvements will better assist other users in understanding the purpose and features of the project. 

Thank you for considering it. 🍻

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,25 @@
 [![Lighthouse PWA Badge](./test_results/lighthouse_pwa.svg)](https://github.com/emazzotta/lighthouse-badges)
 [![Lighthouse SEO Badge](./test_results/lighthouse_seo.svg)](https://github.com/emazzotta/lighthouse-badges)
 
+## Descripcion 
+
 Este proyecto tiene como objetivo crear una API y página web de la Kings League con fines educativos.
+La API proporciona acceso a datos de equipos, presidentes, entrenadores, goleadores y asistentes de la Kings League, y la página web ofrece información sobre los equipos, resultados y clasificaciones de la liga.
+
+La  Api esta construida utilizando "Web scraping" raspado web, hemos utilizado Node.js, Hono y el servicio de hosting de APIs Cloudflare Workers para construir la API. La página web está desarrollada con el framework Astro y utilizamos el framework CSS Tailwind para estilizar la interfaz de usuario. Además, hemos utilizado la librería Cherrio para realizar web scraping y obtener datos de la Kings League.
```
Comment (1fbr) [inline README.md]:
```suggestion
La API  está construída utilizando la técnica de `web scraping`, para ello hemos utilizado Node.js, Hono y el servicio de hosting de APIs Cloudflare Workers. 
La página web está desarrollada con el framework Astro y el framework CSS Tailwind para estilizar la interfaz de usuario. Además, hemos utilizado la librería Cherrio para realizar el `web scraping` y obtener datos de la Kings League.
````

File: README.md [hunk 2]
```diff
@@ -35,8 +46,15 @@ Los siguientes son los endpoints disponibles en la API:
 - GET `/top-assists/:rank`: Devuelve el asistente más destacado de acuerdo a su posicion en la lista de la Kings League.
 - GET `/mvp`: Devuelve los MVPs de la Kings League.
 
-¡Síguenos en nuestro canal de Twitch para ver cómo avanzamos en el desarrollo de la página web!
+## REDES SOCIALES
+
+¡Síguenos en nuestro canal de Twitch para ver cómo avanzamos en el desarrollo de la página web o siguenos en Youtube para mas desarollo web!
```
Comment (1fbr) [inline README.md:51]:
```suggestion
¡Síguenos en nuestro canal de Twitch para ver cómo avanzamos en el desarrollo del proyecto o síguenos en Youtube para más contenido acerca de desarrollo fullstack!
```

File: README.md [hunk 3]
```diff
@@ -35,8 +46,15 @@ Los siguientes son los endpoints disponibles en la API:
...
+
+* [Twitch](https://twitch.tv/midudev)  
+* [YouTube](https://www.youtube.com/c/midudev)
+
+o unete a la comunidad en [Discord](https://discord.gg/midudev)
```
Comment (1fbr) [inline README.md:56]:
```suggestion
o únete a la comunidad en [Discord](https://discord.gg/midudev)
```

File: README.md [hunk 4]
```diff
@@ -12,14 +12,25 @@
...
+## Descripcion 
+
 Este proyecto tiene como objetivo crear una API y página web de la Kings League con fines educativos.
+La API proporciona acceso a datos de equipos, presidentes, entrenadores, goleadores y asistentes de la Kings League, y la página web ofrece información sobre los equipos, resultados y clasificaciones de la liga.
+
+La  Api esta construida utilizando "Web scraping" raspado web, hemos utilizado Node.js, Hono y el servicio de hosting de APIs Cloudflare Workers para construir la API. La página web está desarrollada con el framework Astro y utilizamos el framework CSS Tailwind para estilizar la interfaz de usuario. Además, hemos utilizado la librería Cherrio para realizar web scraping y obtener datos de la Kings League.
+
+Para probar y validar el funcionamiento de la aplicación, hemos utilizado la librería de pruebas ViteTest.
+
+Si quieres ayudarnos, por favor toma un momento para leer el archivo [CONTRIBUTING.md](https://github.com/midudev/kings-league-project/blob/main/CONTRIBUTING.md) Allí encontrarás información útil sobre cómo contribuir de manera efectiva y cómo seguir nuestras guías de estilo. ¡Esperamos que disfrutes colaborando con nosotros!
```
Comment (1fbr) [inline README.md:24]:
```suggestion
Si quieres ayudarnos, por favor toma un momento para leer el archivo [CONTRIBUTING.md](https://github.com/midudev/kings-league-project/blob/main/CONTRIBUTING.md) Allí encontrarás información útil sobre cómo contribuír de manera efectiva y cómo seguir nuestras guías de estilo. ¡Esperamos que disfrutes colaborando con nosotros!
```

**Review by 1fbr — COMMENTED** [review]
Some typos and ideas.

**Review by danyeldev — APPROVED** [review]

**Pull request merged**